src/FastMoney.js

```diff
@@ -3,28 +3,33 @@ import Turn from './Turn';
 import Game from './Game';
 
 class FastMoney extends Round {
-	constructor(question, answers, currentGame, turnNumber, multiplier) {
-		super(question, answers, currentGame, turnNumber)
-		this.player1Guesses = [];
-		this.player2Guesses = [];
-		this.multiplier = multiplier || 1;
-	}
+  constructor(survey, game, multiplier) {
+    super(survey, game)
+    this.player1Guesses = [];
+    this.player2Guesses = [];
+    this.multiplier = multiplier || 1;
+  }
 
-	evaluateGuesses(guess) {
-		this.answers.filter(answer => {
-			if(answer === guess) {
-				this.player1Guesses.push(guess)
-			} 
-		});
-	}
+  evaluateGuesses(guess, turn) {
+    this.answers.filter(answer => {
+      if(answer === guess && turn.player.id === 1) {
+        this.player1Guesses.push(guess)
+      } else if(answer === guess && turn.player.id === 2) {
+        this.player2Guesses.push(guess)
+      }
+      console.log(turn.player.score)
```

Comment:
Try to keep `console.log()`s out of code pushed to master unless there is an absolute need to always see this data in the console for some reason (like the answer to the puzzle/clue).